test/integration/index.js

```diff
@@ -131,9 +130,9 @@ describe('POSTing to /convert', () => {
             para: 'My para'
           }
         })
+        .expect(201)
         .expect('Content-type', /octet-stream/)
-        .expect(res => assert.equal(res.text, result.data))
-        .expect(201);
+        .expect(res => assert.equal(res.text, result));
```

Comment:
looks like this will introduce a breaking change?

Reply:
No, because line 11 of this diff.

In short, the stub object `result` changed because the chrome API changed.

The old module returned `{ data: Buffer }`, but puppeteer returns `Buffer`.